src/Plugin/GraphQL/DataProducer/Entity/EntityUrl.php

```diff
@@ -28,13 +33,16 @@ class EntityUrl extends DataProducerPluginBase {
    * Resolver.
    *
    * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The entity to create a canonical URL for.
+   * @param null|array $options
+   *   The options to provide to the URL generator.
    *
    * @return \Drupal\Core\Url
    *
    * @throws \Drupal\Core\Entity\EntityMalformedException
    */
-  public function resolve(EntityInterface $entity) {
-    return $entity->toUrl();
+  public function resolve(EntityInterface $entity, ?array $options) {
```

Comment:
We discussed this in #1232 if changing the function signature of a dataproducer would be an API break in the stable version of the module.

But since this is such a simple dataproducer I think it is very unlikely that somebody extended this class, so I think we can do this.